packages/web-components/fast-foundation/src/horizontal-scroll/horizontal-scroll.ts

```diff
@@ -368,7 +368,7 @@ export class HorizontalScroll extends FoundationElement {
 
         const current = this.scrollStops.findIndex(
             (stop, index) =>
-                stop <= scrollPosition &&
+                stop >= scrollPosition &&
```

Comment:
@robarbms is there a way we can add a test to ensure this behavior doesn't regress?